web/router.ex

```diff
@@ -43,20 +43,23 @@ defmodule Pairmotron.Router do
   end
 
   scope "/", Pairmotron do
-    pipe_through [:browser, :authenticate] # Use the default browser stack
+    pipe_through [:browser, :authenticate]
 
-    get "/pairs", PairController, :index
     get "/profile", ProfileController, :show
     get "/profile/edit", ProfileController, :edit
     put "/profile/update/:id", ProfileController, :update
     resources "/projects", ProjectController
+
+    get "/pairs", PairController, :index
+    get "/pairs/:year/:week", PairController, :show
+
+    get "/groups/:id/pairs", GroupPairController, :show
+    get "/groups/:id/pairs/:year/:week", GroupPairController, :show
+    delete "/groups/:id/pairs/:year/:week", GroupPairController, :delete
```

Comment:
Why `:id` instead of `:group_id`? As these are routes for `Pair`s, I'd think `:id` would be reserved for a specific pair (if there was ever some route for updating a specific pair for some reason). Therefore, I'd tihnk that `:group_id` would be more appropriate here for the route.